testcases/crypto/abfunc.c

```diff
@@ -1180,7 +1180,7 @@ int main(int argc, char **argv)
     }
 
     if (is_ep11_token(SLOT_ID)) {
-        testcase_setup(0);
+        testcase_setup();
         rc = do_CheckMechanismInfo();
```

Comment:
Yes, that's true. Are you also removing the arg from the macro definition in regress.h?

Reply:
sure